### DISCUSSION TITLE
System Requirements/Raspberry Pi

### DISCUSSION THREAD

**Discussion by petebocken** [teslamate-org/teslamate#1701] [Q&A] [answered]
I’ve had Teslamate installed on my Windows PC in Docker for the past year. I’d like to migrate it over to a new dedicated Pi. I can’t find any system requirements anywhere. I did hear it won’t run on a 0W, so figured 3 or 4. Anyone running theirs on a Pi?

**Comment by petebocken** [comment, marked as answer]
I setup a 3 Model B+ and working fine so far.
https://i.imgur.com/Q90PMAQ.jpg

**Reply by amcypher** [reply]
I use 3 Model B+ as well - Teslamate uses about 20% memory on it at all times - CPU load is non existent unless starting/stopping Teslamate.

**Reply by petebocken** [reply]
Which OS? I’m using Raspian Lite with Docker and I sit around 50%. I’m guessing cause of Docker?

**Reply by Tiz-B** [reply]
Does it work on Raspberry Pi2?
